inst/rmarkdown/resources/css/customize-bs4.css

```
@@ -0,0 +1,96 @@
+@media screen {
+  .container-fluid {
+    padding-right: 0;
+    padding-left: 0;
+  }
+  h1:before {
+    display: none;
+  }
+}
+
+
+.book-part{
+  color: #93bcbc;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.li a {
+  color: #cf581b;
+}
+
+/* Ecraser les réglages de bs4 */
+@media print {
+  @page {
+    size: 210mm 297mm;
+  }
+  html, body {
+    line-height: 1.3;
+    widows: 2;
+    orphans: 2;
+  }
+  pre {
+    background-image: none;
+    border: none;
+  }
+  /* Masquer des choses */
+  button[id="print"],
```

C'est plus efficace d'écrire 
```suggestion
  #print,
```